src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/ForgotPassword.razor.cs

```diff
@@ -53,18 +53,12 @@ private bool ValidateForgotPassword()
 
     private async Task Submit()
     {
-        if (IsLoading)
+        var isForgotPasswordInvalid = ValidateForgotPassword() is false;
```

Comment:
why did you define a new variable here?
I think it's redundant. the `ValidateForgotPassword() is false` is more expressive.

Reply:
I thought this is more cleaner and more readable

Reply:
it's not, plz remove all of these variables.

Reply:
originally, the expression was `!ValidateForgotPassword()` but because using the NOT operator at the start of a condition expression is not clear enough, we decided to replace it with a direct comparison with `false` in this project. but introducing another variable for this new expression is adding another level of complexity and also is not as clear as before.